src/client/manifest.json

```diff
@@ -1,26 +1,16 @@
 {
-  "name": "Hacker News",
-  "short_name": "HN",
+  "name": "nearForm Hacker News",
+  "short_name": "nearForm HN",
   "icons": [
     {
-      "src": "/images/hn-logo-192x192.png",
-      "sizes": "192x192",
-      "type": "image/png"
-    },
-    {
-      "src": "/images/hn-logo-256x256.png",
-      "sizes": "256x256",
-      "type": "image/png"
-    },
-    {
-      "src": "/images/hn-logo-512x512.png",
-      "sizes": "512x512",
+      "src": "/images/favicon-1024.png",
+      "sizes": "1024x1024",
       "type": "image/png"
     }
   ],
   "start_url": "/",
   "display": "standalone",
   "orientation": "portrait",
-  "background_color": "#ff6600",
-  "theme_color": "#ff6600"
-}
\ No newline at end of file
+  "background_color": "#ec1c2b",
```

Comment:
ideally these would come from the theme file which would require the manifest to be built (unfortunately adding complexity)

Reply:
probably a good idea to add a version key to the manifest to provide cache updating abilities in case of changes
https://developer.chrome.com/extensions/manifest/version

Reply:
I've left this as is for now, it doesn't seem worth the extra complexity I don't think.

In regards to versioning we currently have a `max-age=0` header on the response and we're caching it with the service worker, this should handle the case where when the SW updates the manifest file will be refetched but if the SW hasn't been updated the request is served from the SW so the user isn't making unnecessary network requests.